app/helpers/peer_review_helper.rb

```diff
@@ -8,8 +8,9 @@ def create_map_reviewee_to_reviewers(reviewer_groups, reviewee_groups)
     peer_review_map = Hash.new { |hash, key| hash[key] = [] }
     reviewee_groups.each { |reviewee| peer_review_map[reviewee['id']] }
 
-    peer_reviews = PeerReview.where(reviewer_id: reviewer_ids)
-    peer_reviews.each do |peer_review|
+    PeerReview.includes(:reviewer)
```

Comment:
I don't think you need the `reviewer` includes; you could actually do `peer_review.reviewer_id` below